docs/docs/reference/new-types/match-types.md

```diff
@@ -103,7 +103,7 @@ The first rule is a structural comparison between two match types:
 ```
   S <: T,  m <= n,  Ci <: Di for i in 1..n
 ```
-I.e. scrutinees and corresponding cases must be subtypes, no case re-ordering is allowed, but the subtype can have more cases than the supertype.
```

Comment:
This "yet not less" doesn't give more meaning to this line I think.

Reply:
I agree that it is not as important as `the subtype can have more cases` which is implicit information as well (looking at the subtype rule) -- that's why I have put it into parentheses. But I would keep it to make another implicit information explicit. But as you wish, I don't mind to remove it again.

Reply:
Yeap, I think its not needed